tools/schema_registry_tools/schemas_command_generator.py

```diff
@@ -59,21 +151,18 @@ def generate_options():
     tmp = re.sub('},$', '}]', tmp)
     j = json.loads(tmp)
```

Comment:
I think it would help me if there was example input file somewhere. And maybe the command to generate it.